kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverReportConfig.kt

```diff
@@ -0,0 +1,1225 @@
+     * ```
+     *  binary {
+     *      filters {
+     *          // ...
+     *      }
+     *
+     *      onCheck.set(false)
+     *      file.set(layout.buildDirectory.file("my-project-report/report.ic"))
+     *  }
+     * ```
+     *
+     * Kover binary report is compatible with IntelliJ Coverage report (ic)
+     */
+    public fun binary(config: Action<KoverBinaryTaskConfig>)
+
+    /**
+     * Configure coverage verification for current report variant.
+     * ```
+     *  verify {
+     *      onCheck = true
+     *
+     *      rule {
+     *          // ...
+     *      }
+     *
+     *      rule("Custom Name") {
+     *          // ...
+     *      }
+     *  }
+     * ```
+     */
+    public fun verify(config: Action<KoverVerifyTaskConfig>)
+
+    /**
+     * Configure coverage verification for current report variant.
+     * ```
+     *  verify {
+     *      onCheck = true
+     *
+     *      rule {
+     *          // ...
+     *      }
+     *
+     *      rule("Custom Name") {
+     *          // ...
+     *      }
+     *  }
+     * ```
+     */
+    public fun verifyAppend(config: Action<KoverVerifyTaskConfig>)
+
+    /**
+     * Configure coverage printing to the log for current report variant.
+     * ```
+     *  log {
+     *      onCheck = true
+     *
+     *      filters {
+     *          // ...
+     *      }
+     *      header = null
+     *      format = "<entity> line coverage: <value>%"
+     *      groupBy = GroupingEntityType.APPLICATION
+     *      coverageUnits = MetricType.LINE
+     *      aggregationForGroup = AggregationType.COVERED_PERCENTAGE
+     *  }
+     * ```
+     */
+    public fun log(config: Action<KoverLogTaskConfig>)
+}
+
+/**
+ * Configuration of coverage printing to the log for current report variant.
+ * ```
+ *  log {
+ *      onCheck = true
+ *
+ *      filters {
+ *          // ...
+ *      }
+ *      header = null
+ *      format = "<entity> line coverage: <value>%"
+ *      groupBy = GroupingEntityType.APPLICATION
+ *      coverageUnits = MetricType.LINE
+ *      aggregationForGroup = AggregationType.COVERED_PERCENTAGE
+ *  }
+ * ```
+ */
+public interface KoverLogTaskConfig {
+    @Deprecated(
+        message = "It is forbidden to override filters for a specific report, use custom report variants to create reports with a different set of filters. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
+        level = DeprecationLevel.ERROR
+    )
+    public fun filters(config: Action<KoverReportFiltersConfig>) {
+        throw KoverDeprecationException("It is forbidden to override filters for the log report, use custom report variants to create reports with a different set of filters. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+    }
+
+    /**
+     * Print coverage when running the `check` task.
+     *
+     * Empty by default.
```

Comment:
And what does `empty` mean? Will it run with the `check` task or not? Please add this to the doc

Reply:
And how Boolean can be empty? Maybe you mean `true`?

Reply:
> And how Boolean can be empty? 

The property is of type `Property`.
For this type, the value can be either present (isPresent) or absent. Perhaps the word `absent` is more appropriate than `empty`.

Reply:
> Please add this to the doc

OK